pkg/cli/build-minirootfs.go

```diff
@@ -64,6 +68,29 @@ func BuildMinirootFSCmd(ctx context.Context, configFile string, outputTarGZ stri
 		WorkDir:            wd,
 		TarballPath:        outputTarGZ,
 		UseProot:           useProot,
+		SourceDataEpoch:    time.Now(),
+	}
+
+	if v, ok := os.LookupEnv("SOURCE_DATE_EPOCH"); ok {
+		// The value MUST be an ASCII representation of an integer
+		// with no fractional component, identical to the output
+		// format of date +%s.
+		sec, err := strconv.ParseInt(v, 10, 64)
+		if err != nil {
+			// If the value is malformed, the build process
+			// SHOULD exit with a non-zero error code.
+			return err
+		}
+
+		bc.SourceDataEpoch = time.Unix(sec, 0)
+	} else if buildDate == "now" {
+		bc.SourceDataEpoch = time.Now()
+	} else {
+		sec, err := strconv.ParseInt(v, 10, 64)
```

Comment:
What's the format for the build-date flag ? I'm using the same logic for `SOURCE_DATE_EPOCH` (from the spec: The `value MUST be an ASCII representation of an integer with no fractional component, identical to the output format of date +%s.`) but it's probably better to adopt a more user-friendly format for this flag.

Reply:
Probably the ISO format outlined in the bug, support both ideally.

Reply:
Parsing the string as RFC3339